### ISSUE TITLE
Documentation needs updating

### ISSUE THREAD

**Issue by bobonthenet** [Gainsight/px-react-native-sdk#17]
The documentation is extremely difficult to follow and has information that is incorrect. Please review and correct the documentation at https://support.gainsight.com/PX/Mobile/Mobile_Platforms/Install_Gainsight_PX_React-Native

1. There is no .tgz file at the provided github link.
2. The iOS steps appear to give different methods for performing the same process as separate steps.
3. The rest is just hard to follow.

**Comment by hexadecy** [comment]
It should be updated to have at least:
```shell
npm install @gainsight-px/react-native-gainsight-px --save
```
